ASTProcessor/src/main/java/edu/usb/argos/ASTProcessor/application/analyzers/IDirectoryAnalyzer.java

```diff
@@ -0,0 +1,7 @@
+package edu.usb.argos.ASTProcessor.application.analyzers;
+
+import java.util.List;
+
+public interface IDirectoryAnalyzer<TSourceCode, TAst> {
```

Comment:
I think that our Trainer said, that don't use that, only use `T`, `F`, etc

Reply:
done